ONSdigital/dp-frontend-renderer
# Add frontend model to dataset landing page and preview/download page

### What

Added frontend model information into Filterable Dataset landing page and Preview/Download page

### How to review

- Run this service on this branch, as well as `dp-frontend-dataset-controller` on feature/datasetlandingpage and `dp-frontend-filter-dataset-controller` on feature/preview-page-api-to-model
- Follow the user journey from the dataset landing page through the CMD journey to Preview and Download
- Verify that the landing page and preview page both render correctly and have the correct data (bearing in mind the expected api responses have been stubbed out in the code)

### Who can review

Anyone
